src/Reflection/AbstractFunction.php

```diff
@@ -364,64 +349,46 @@ public function __get($key)
      * @param mixed $value
      * @return void
      */
-    public function __set($key, $value)
+    public function __set(string $key, $value): void
     {
         $this->config[$key] = $value;
     }
 
     /**
      * Set method's namespace
      *
-     * @param string $namespace
      * @throws Exception\InvalidArgumentException
-     * @return void
      */
-    public function setNamespace($namespace)
+    public function setNamespace(?string $namespace = null): void
     {
         if (empty($namespace)) {
```

Comment:
Because of that, maybe we should keep namespace as `string` and default (in properties) as empty string?
I think it was like that before. I am fine also with `null` which means that is not set/not defined, but it should behave the same as empty string probably...

Reply:
Yes, namespace can be optional. Not set is represented by `null`.